common-content/en/module/logic/introduction/index.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "Introduction"
+time = 5
+emoji= "🧠"
+[build]
+render = 'never'
+list = 'local'
+publishResources = false
++++
+
+Welcome to Logic. For this module, we will get out of VSCode and build mental models we can use anywhere to reason more effectively.
+
+### You will need
+
+📓 Notebook and pen  
+🧠 Your wonderful brain  
+🔍 And you will still need the curriculum and Google, that's not banned, don't worry.
+
+### Learning by teaching
```

Comment:
I assume this will come but: It's not clear to me from the introduction how much the trainees should be learning each of these models. Should they be learning, understanding, and playing a game for _each_ of them? Or focusing on getting one really well, and learning the others through the workshop? Or somewhere in the middle?

Reply:
For each. They need all these pieces plus base case and binary to solve the main puzzle.

Reply:
Help me phrase that?

Reply:
I made a wording suggestion in a separate comment - let's discuss the specifics there? :)